src/Dist_Archive_Command.php

```diff
@@ -513,4 +515,82 @@ private function get_file_list( $source_dir_path, $excluded = false ) {
 
 		return $excluded ? $excluded_files : $included_files;
 	}
+
+	/**
+	 * Converts a number of bytes to the largest unit the bytes will fit into.
+	 *
+	 * @param int $bytes    Number of bytes.
+	 * @param int $decimals Precision of number of decimal places.
+	 * @return string Number string.
+	 */
+	private function get_size_format( $bytes, $decimals = 0 ) {
```

Comment:
As per my issue comment, we should consider contributing this to the main WP-CLI utils

Reply:
OK I see you opened an issue for that now.

Reply:
This is pretty basic version for size format. Code mentioned in the DB is pretty much tied to database arguments. So, when we add this method in Utils, there would more arguments and customization.

Reply:
Should we work on that before merging this PR? Or keep this here now? This is private method so, I guess it wont be difficult to replace here when main function in merged in core repo.

Reply:
Fair, let's keep it as-is for now.